Resources/Maps/Shuttles/Mushroom.yml

```diff
@@ -0,0 +1,5827 @@
+  entities:
+  - uid: 72
+    components:
+    - pos: 3.5,-1.5
+      parent: 1
+      type: Transform
+    - locked: False
+      type: Lock
+    - air:
+        volume: 200
+        immutable: False
+        temperature: 293.1496
+        moles:
+        - 1.8856695
+        - 7.0937095
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+      type: EntityStorage
+    - containers:
+        entity_storage: !type:Container
+          showEnts: False
+          occludes: True
+          ents:
+          - 73
+          - 75
+          - 74
+          - 76
+        paper_label: !type:ContainerSlot
+          showEnts: False
+          occludes: True
+          ent: null
+      type: ContainerContainer
+- proto: Gyroscope
+  entities:
+  - uid: 575
+    components:
+    - rot: 1.5707963267948966 rad
+      pos: -3.5,-20.5
+      parent: 1
+      type: Transform
+- proto: HighSecCaptainLocked
+  entities:
+  - uid: 576
+    components:
+    - pos: 2.5,-2.5
+      parent: 1
+      type: Transform
+  - uid: 577
+    components:
+    - pos: -4.5,-2.5
+      parent: 1
+      type: Transform
+- proto: HospitalCurtainsOpen
+  entities:
+  - uid: 708
+    components:
+    - rot: -1.5707963267948966 rad
+      pos: 1.5,-10.5
+      parent: 1
+      type: Transform
+  - uid: 866
+    components:
+    - rot: -1.5707963267948966 rad
+      pos: -3.5,-10.5
+      parent: 1
+      type: Transform
+- proto: IntercomSecurity
+  entities:
+  - uid: 436
+    components:
+    - pos: 0.5,1.5
+      parent: 1
+      type: Transform
+- proto: KitchenMicrowave
+  entities:
+  - uid: 578
+    components:
+    - pos: -3.5,-5.5
+      parent: 1
+      type: Transform
+- proto: LockerCaptain
+  entities:
+  - uid: 398
+    components:
+    - pos: -5.5,-1.5
+      parent: 1
+      type: Transform
+    - locked: False
+      type: Lock
+    - fixtures:
+        fix1:
+          shape: !type:PolygonShape
+            radius: 0.01
+            vertices:
+            - -0.25,-0.48
+            - 0.25,-0.48
+            - 0.25,0.48
+            - -0.25,0.48
+          mask:
+          - Impassable
+          - MidImpassable
+          - LowImpassable
+          layer:
+          - BulletImpassable
+          - Opaque
+          density: 75
+          hard: True
+          restitution: 0
+          friction: 0.4
+      type: Fixtures
+    - air:
+        volume: 200
+        immutable: False
+        temperature: 293.1496
+        moles:
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+      open: True
+      removedMasks: 20
+      type: EntityStorage
+    - isPlaceable: True
+      type: PlaceableSurface
+- proto: LockerParamedicFilledHardsuit
+  entities:
+  - uid: 579
+    components:
+    - pos: 0.5,-5.5
+      parent: 1
+      type: Transform
+- proto: LockerSalvageSpecialistFilledHardsuit
```

Comment:
Salvage locker also?

Reply:
all expedition ships that i have send has an salvage locker but the anchor and the dart that i have seen?

Reply:
this is fine dw